src/JsonResponses.cpp

```diff
@@ -130,9 +131,24 @@ namespace JsonResponses {
     return ss.str();
   }
 
-  int64_t getTimeStamp(){
-    return std::chrono::duration_cast<std::chrono::milliseconds>(
-          std::chrono::system_clock::now().time_since_epoch()
-          ).count();
+/** Return an extended ISO8601 UTC timestamp according to W3C guidelines https://www.w3.org/TR/NOTE-datetime
+ * Complete date plus hours, minutes, seconds and a decimal fraction of a second
+      YYYY-MM-DDThh:mm:ss.sTZD (eg 1997-07-16T19:20:30.45+01:00)
+    
+    where:
+
+     YYYY = four-digit year
+     MM   = two-digit month (01=January, etc.)
+     DD   = two-digit day of month (01 through 31)
+     hh   = two digits of hour (00 through 23) (am/pm NOT allowed)
+     mm   = two digits of minute (00 through 59)
+     ss   = two digits of second (00 through 59)
+     s    = one or more digits representing a decimal fraction of a second
+     TZD  = time zone designator (Z or +hh:mm or -hh:mm)
+*/
+  std::string getTimeStamp(){
+    using namespace boost::posix_time;
+    ptime t = microsec_clock::universal_time();
+    return to_iso_extended_string(t)+"Z";
```

Comment:
I am not a fan to use boost library if any implementation already exists in std lib. Std lib is hardware independent and more stable. Better like this?:
```
  auto now = std::chrono::system_clock::now();
  auto itt = std::chrono::system_clock::to_time_t(now);
  std::ostringstream ss;
  ss << std::put_time(gmtime(&itt), "%FT%T.%sZ");
```
https://techoverflow.net/2018/03/30/iso8601-utc-time-as-stdstring-using-c11-chrono/